File: core/src/main/java/ru/surfstudio/android/core/app/interactor/common/network/error/CacheEmptyException.java
package ru.surfstudio.android.core.app.interactor.common.network.error;

/**
 * внутренне исключение для механизма работы с сервером, используется для возвращения null в случае
 * попытки получения кеша, которого не существует
 */
public class CacheEmptyException extends NetworkException {
}

File: template/src/main/java/ru/surfstudio/standard/interactor/common/error/CacheEmptyException.kt
package ru.surfstudio.standard.interactor.common.error

/**
 * внутренне исключение для механизма работы с сервером, используется для возвращения null в случае
 * попытки получения кеша, которого не существует
 */
class CacheEmptyException : NetworkException()
